packages/features/calendars/CalendarSwitch.tsx

```diff
@@ -47,12 +47,11 @@ const CalendarSwitch = (props: ICalendarSwitchProps) => {
           throw new Error("Something went wrong");
         }
       } else {
-        const res = await fetch("/api/availability/calendar", {
+        const res = await fetch("/api/availability/calendar?" + new URLSearchParams(body), {
           method: "DELETE",
           headers: {
             "Content-Type": "application/json",
           },
-          body: JSON.stringify(body),
         });
```

Comment:
@emrysal I'm wondering why is this needed? is this related to the NextJS update?

Reply:
See my comment on this PR that elaborates why; tl;dr; yes.

Reply:
Ooh! I got it, thanks!